.github/workflows/coverage.yml

```diff
@@ -0,0 +1,66 @@
+name: Tests Coverage
+
+on:
+  push:
+    branches:
+      - coverage_test
+    paths-ignore:
+      - "**/dependabot.yml"
+      - "**/README.md"
+  pull_request:
+    branches:
+      - dev
+  workflow_dispatch:
+
+env:
+  TARPAULIN_VERSION: 0.20.1
```

Comment:
- not used
- we can remove unused commands

Reply:
removed.